leaf-ui/js/displayAnalysis.js

```diff
@@ -19,7 +19,8 @@ var currAnalysisConfig;
 function displayAnalysis(analysisResults){
```

Comment:
Could you update the docstring of this to specify `@param {AnalysisResult} analysisResult`? That way it makes it clear that this function is being passed an AnalysisResult object

Reply:
same should be true for any other method that is taking in the currentAnalysis object as a param - maybe walk through displayAnalysis.js and update all docstrings